yb_migrate/src/migration/postgres.go

```diff
@@ -50,7 +50,11 @@ func PgDumpExtractSchema(source *srcdb.Source, exportDir string) {
 	log.Infof("Running command: %s", preparePgdumpCommandString)
 	preparedYsqldumpCommand := exec.Command("/bin/bash", "-c", preparePgdumpCommandString)
 
-	err := preparedYsqldumpCommand.Run()
+	stdout, err := preparedYsqldumpCommand.CombinedOutput()
+	//pg_dump formats its stdout messages, %s is sufficient.
+	if string(stdout) != "" {
+		log.Infof("%s", string(stdout))
+	}
```

Comment:
Can we use a mechanism similar to other places i.e. the one that uses `bytes.Buffer` for STDOUT and STDERR.

Reply:
The reason I have not opted for `bytes.Buffer` over here (and other places) is because the command is/was being executed like so: `command.Run()`, which does not need a corresponding `command.Wait()`. Wherever there is/was a `command.Start()', we can choose to opt for the `bytes.Buffer` approach. Instead I have replaced `command.Run()` with `command.CombinedOutput()`, which provides both STDOUT and STDERR, while also not needing to call `command.Wait()`